test/TestApps/MsBuildTestApp/MsBuildTestApp/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0"
+    },
+    "Microsoft.NETCore.Sdk": {
+      "version": "1.0.0-alpha-20160902-5"
+    }    
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "imports": "portable-net45+win8+wp8+wpa81"
```

Comment:
`Microsoft.Composition` still needs it. 
It is coming through the Roslyn packages. I think once I update those packages this will be gone.